website/docs/docs/build/metrics-overview.md

```diff
@@ -11,13 +11,52 @@ Once you've created your semantic models, it's time to start adding metrics! Met
 
 The keys for metrics definitions are:
 
+<!-- for v1.8 and higher -->
+
+<VersionBlock firstVersion="1.8">
+
+| Parameter | Description | Type |
+| --------- | ----------- | ---- |
+| `name` | Provide the reference name for the metric. This name must be unique amongst all metrics.   | Required |
+| `description` | Describe your metric.   | Optional |
+| `type` | Define the type of metric, which can be `simple`, `ratio`, `cumulative`, or `derived`.  | Required |
+| `type_params` | Additional parameters used to configure metrics. `type_params` are different for each metric type. | Required |
+| `config` | Provide the specific configurations for your metric.   | Optional |
+| `config::meta` | Use the [`meta` config](/reference/resource-configs/meta) to set metadata for a resource.  | Optional |
```

Comment:
Looks great overall but where did this `config::meta` format come from? Should there be a line break?